crates/sui-indexer-alt/src/handlers/coin_balance_buckets.rs

```diff
@@ -58,7 +60,7 @@ impl Processor for CoinBalanceBuckets {
 
     fn process(&self, checkpoint: &Arc<CheckpointData>) -> Result<Vec<Self::Value>> {
         let cp_sequence_number = checkpoint.checkpoint_summary.sequence_number;
-        let checkpoint_input_objects = checkpoint.checkpoint_input_objects();
```

Comment:
I did not expect coin balance bucket to be affected too! Of course this is only affected if the coin is a shared object which is rare enough.